paasta_tools/utils.py

```diff
@@ -3091,6 +3092,13 @@ def get_pipeline_config(service: str, soa_dir: str = DEFAULT_SOA_DIR) -> List[Di
     return service_configuration.get("deploy", {}).get("pipeline", [])
 
 
+def is_secrets_for_teams_enabled(service: str, soa_dir: str = DEFAULT_SOA_DIR) -> bool:
+    service_yaml_contents = read_extra_service_information(service, "service", soa_dir)
+    if "secrets_for_owner_team" in service_yaml_contents:
+        return service_yaml_contents["secrets_for_owner_team"]
+    return False
```

Comment:
nit, this is effectively 
```suggestion
    return service_yaml_contents.get("secrets_for_owner_team", False)
```